test/cpp/end2end/time_change_test.cc

```diff
@@ -129,25 +129,34 @@ class TimeChangeTest : public ::testing::Test {
  protected:
   TimeChangeTest() {}
 
-  void SetUp() {
+  static void SetUpTestCase() {
```

Comment:
How is this function run?

Reply:
`SetUpTestCase` is run once before any testcases in the test fixture are executed (it's executed once per test fixture). `SetUp` in run before every testcase.
It's documented [here](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#sharing-resources-between-tests-in-the-same-test-suite), but note that the name of the static function mentioned in the documentation (`SetUpTestSuite`) is different.  That's because `SetUpTestCase` was recently renamed to `SetUpTestSuite`, but the version of googletest that we're using still uses `SetUpTestCase`.

Reply:
sg. thanks!